### DISCUSSION TITLE
Is differential privacy is enough to remove copy problem and bundling problem ?

### DISCUSSION THREAD

**Discussion by manik-500** [OpenMined/courses#115] [Course: Our Privacy Opportunity] [answered]
I think differential privacy is enough to remove copy problem and bundling problem because we are adding random noise on input so nobody actually can copy my real data.
Am i right or some cases it can be wrong ?

**Comment by yemikifouly** [comment, marked as answer]
Hello Manik! 👋 

@iamtrask or @em-blue Could you help please?

**Reply by em-blue** [reply]
@manik-500 Have you finished Lesson 4 yet? :-)

**Reply by manik-500** [reply]
I have just finished lesson 4 . 
I have got my answer differential privacy will not remove copy problem rather than bundling problem.
thanks.
